orbit_determination/od_simulation_data_manager.py

```diff
@@ -38,11 +38,14 @@ class ODSimulationDataManager:
 
     states: np.ndarray = field(default_factory=lambda: np.zeros(shape=(0, 6)))
     eci_Rs_body: np.ndarray = field(default_factory=lambda: np.zeros(shape=(0, 3, 3)))
-
+      
     measurement_indices: np.ndarray = field(default_factory=lambda: np.array([], dtype=int))
     bearing_unit_vectors: np.ndarray = field(default_factory=lambda: np.zeros(shape=(0, 3)))
     landmarks: np.ndarray = field(default_factory=lambda: np.zeros(shape=(0, 3)))
 
+    curr_bearing_unit_vectors: np.ndarray = field(default_factory=lambda: np.zeros(shape=(0, 3)))
```

Comment:
With the suggestions in `test_ekf.py`, none of the (non-formatting) changes in this file should be necessary.